apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts

```diff
@@ -12,7 +12,7 @@ export const useBoostedPoolApr = ({ contractAddress, chainId }: UseBoostedPoolAp
   const client = getViemClients({ chainId })
 
   const { data } = useQuery(
-    ['boostedPoolsApr'],
+    ['boostedPoolsApr', client, contractAddress, chainId],
```

Comment:
can remove client? As it's bound with chainId

Reply:
Done

Reply:
Done